bootstrapvz/providers/ec2/__init__.py

```diff
@@ -80,12 +80,13 @@ def resolve_tasks(taskset, manifest):
         taskset.add(tasks.network.EnableDHCPCDDNS)
 
     if manifest.release >= jessie:
-        taskset.add(apt.AddBackports)
         taskset.add(tasks.tuning.SetCloudInitMountOptions)
         taskset.add(tasks.packages.AddWorkaroundGrowpart)
         taskset.add(initd.AdjustGrowpartWorkaround)
         if manifest.system['bootloader'] == 'grub':
             taskset.add(grub.EnableSystemd)
+        if manifest.release == stable:
```

Comment:
I think I prefer to stick to stable for simple reason in the future we may need backports for the future stable and this would require changes in the code again, with what I'm suggesting we're always having it available if needed.